backend/src/main/java/com/team28/sidedish/service/CategoryService.java

```diff
@@ -1,10 +1,23 @@
 package com.team28.sidedish.service;
 
 import com.team28.sidedish.controller.dto.CategoryResponse;
+import com.team28.sidedish.repository.CategoryRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.context.annotation.Primary;
+import org.springframework.stereotype.Service;
 
 import java.util.List;
+import java.util.stream.Collectors;
 
-public interface CategoryService {
+@Service
+@RequiredArgsConstructor
+public class CategoryService {
 
-    List<CategoryResponse> findAll();
+    private final CategoryRepository categoryRepository;
+
+    public List<CategoryResponse> findAll() {
+        return categoryRepository.findAll().stream()
+                .map(categoryEntity -> CategoryResponse.of(categoryEntity.getId(), categoryEntity.getName()))
```

Comment:
```suggestion
                .map(categoryEntity -> CategoryResponse.from(categoryEntity)
```

위처럼 아규먼트의 개수를 줄여도 되지 않을까요?